src/Database/Type/DateType.php

```diff
@@ -49,27 +47,13 @@ class DateType extends DateTimeType
     protected $setToDateStart = true;
 
     /**
-     * Change the preferred class name to the FrozenDate implementation.
-     *
-     * @return $this
-     */
-    public function useImmutable()
```

Comment:
If we aren't deprecating this in 4.3 (as we shouldn't) then we should also consider leaving this method in as a no-op to avoid breaking compatibility. We'll have guided people to call this method in every application they have and it will be an annoying bump to have to fix immediately.

Reply:
Hmm... I think it's best to deprecated this method too in 4.3. It's called from the constructor itself so generally users don't need to call it explicitly.